Providers/nxOMSAutomationWorker/automationworker/worker/omsutil.py

```diff
@@ -41,11 +41,20 @@ def initialize():
     if os.getuid() == nxautomation_uid:
         omsagent_group = grp.getgrnam(GROUPNAME_OMSAGENT)
         if USERNAME_NXAUTOMATION not in omsagent_group.gr_mem:
+            print_success_message = False
```

Comment:
I don't think this logic (with print_success_message) is needed. In one case an exception is raised else we can print the sucess message.
I don't have time to fix this now but let's keep a note and refactor for next release.
Thanks for making the fix!